src/python/WMCore/Services/DBS/DBSErrors.py

```diff
@@ -96,6 +96,20 @@ def getServerCode(self):
             return self.data['error']['code']
         return 0
 
+    def getCodes(self):
```

Comment:
I don't see this method being used anywhere other than in the unit tests. If this is no longer relevant, I would suggest to actually remove it.
If it is used only inside the test, then I think it would be better to move this implementation out of the `src/` directory (likely in the test/ one).

Reply:
fine, removed